config/environments/development.rb

```diff
@@ -62,4 +62,8 @@
 
 	# Uncomment if you wish to allow Action Cable access from any origin.
 	# config.action_cable.disable_request_forgery_protection = true
+
+	# config jwt_secret_key
+	config.jwt_secret_key = 'ch6jdh74wg460jh4dge6jfgyt63jt24e'
+	config.jwt_token_expire_time = 5.days
```

Comment:
Whaf if we have only `5` or wrap expire time like this
```
config.jwt_token_expire_time = -> { 5.days.after }
```

Reply:
Another thought what if we setup $JWT_EXPIRE_TIME in .env

Reply:
.env in this case, this is a completely unnecessary thing in the development environment, especially since a few years ago a better method appeared, which I use. this can be seen in the file for the production environment